pkg/sdk/api_integrations_def.go

```diff
@@ -151,7 +151,9 @@ var ApiIntegrationsDef = g.NewInterface(
 			SQL("API INTEGRATIONS").
 			OptionalLike(),
 	).
-	ShowByIdOperation().
+	ShowByIdOperationWithFiltering(
```

Comment:
Nit: pls run the generator (or adjust the output files manually) for the changed definitions. I mean that we should probably expect some newline changes.

Reply:
done